modules/gcp_bigquery_view/inputs.tf

```diff
@@ -0,0 +1,19 @@
+variable "dataset_id" {
+  type        = string
+  description = "The dataset ID to create the view in. Changing this forces a new resource to be created."
+}
+
+variable "name" {
+  type        = string
+  description = "A view name for the resource. Changing this forces a new resource to be created."
+}
+
+variable "description" {
+  type        = string
+  description = "The field description."
```

Comment:
This should be description of what the view does.

Reply:
Any suggestion how to word it? This is what is used in the [GCP terraform provider](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/bigquery_table#description) and what [we use for tables](https://github.com/honestbank/terraform-gcp-bigquery/blob/main/modules/gcp_bigquery_table/inputs.tf#L13).

Reply:
`The description of what the view does` should be good.